api/options/options.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 Cisco Systems, Inc. and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "google/protobuf/descriptor.proto";
+
+package istio_operator.v2.api.options;
+
+option go_package = "github.com/banzaicloud/istio-operator/api/v2/options;options";
```

Comment:
just checking: `options;options` is intended to be like this?

Reply:
it was intended, but `;options` is not needed, so it got removed.